bazarr/sonarr/sync/parser.py

```diff
@@ -78,7 +78,11 @@ def seriesParser(show, action, tags_dict, language_profiles, serie_default_profi
         tag_profile = get_matching_profile(tags, language_profiles)
         if tag_profile:
             parsed_series['profileId'] = tag_profile
-    
+        remove_profile_tags_list = settings.general.remove_profile_tags
+        if len(remove_profile_tags_list) > 0:
+            if set(tags) & set(remove_profile_tags_list):
+                parsed_series['profileId'] = None
```

Comment:
This is just a question about the behavior, once it matches this once, the profile is permanent removed?

Reply:
Nvm this comment, all records are updated on the database during the re-index.

Reply:
No, only the association between that profile and the video is removed.
The profile itself is untouched.
The association can be easily restored by editing the video and selecting a profile.